Prox/Prox/PlaceCarousel/PlaceCarouselViewController.swift

```diff
@@ -360,3 +358,26 @@ extension PlaceCarouselViewController: LocationProvider {
     }
 }
 
+extension PlaceCarouselViewController: PlacesControllerDelegate {
+    func placesControllerNoPlacesFound(_ controller: PlacesController, willRetry retry: Bool) {
+        if retry {
+            // TODO placeholder for the waiting state.
+            headerView.numberOfPlacesLabel.text = "Waiting"
+        } else {
+            self.places = []
```

Comment:
add `self.placeCarousel.refresh()` else the placeCarousel will never update to the new value